src/index.html

```diff
@@ -1,17 +1,175 @@
-<!doctype html>
+<!Doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script type="text/javascript" src="./main.js">
+    </script>
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" 
+            name="surname" 
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label class="form-field">
+          Name:
+          <input type="text" name="name">
+        </label>
+
+
+        <label class="form-field">
+          How old are You?
+          <input type="number" 
+            name="age" 
+            min="1" 
+            max="100"
+            value="12"
+          >
+        </label>
+
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" 
+            name="trip-start" 
+            value="12-31-2000" 
+            min="01-01-2000" 
+            max="12-31-2200" 
+            required
+          >
+        </label>
+
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" 
+            name="remember" 
+            required
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input type="email" 
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+
+        <label class="form-field">
+          Password:
+          <input type="password" 
+            name="password" 
+            minlength="6" 
+            maxlength="10" 
+            required
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
+          Do you love cats?
```

Comment:
remove this text from input label, and move above `<label>`
```suggestion
          Do you love cats?
          <label>
```